boards/beagle/pocketbeagle_2/pocketbeagle_2_am6232_m4.yaml

```diff
@@ -0,0 +1,8 @@
+identifier: pocketbeagle_2/am6232/m4
+name: BeagleBoard.org PocketBeagle 2
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+ram: 192
```

Comment:
having supported / uart entry here would be good to ensure coverage in CI. but can be addressed later i guess

Reply:
Good catch. Added it.